Common/DataModel/EseTable.h

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+// q vector framework with ESE (20/08/2024)
+//
+/// \author Joachim Hansen <joachim.hansen@cern.ch>
+//
+
+#ifndef COMMON_DATAMODEL_ESETABLE_H_
+#define COMMON_DATAMODEL_ESETABLE_H_
+
+#include <vector>
+
+#include "Framework/AnalysisTask.h"
+#include "Framework/ASoAHelpers.h"
+
+namespace o2::aod
+{
+namespace qVector
```

Comment:
```suggestion
namespace q_vector
```
Please respect naming conventions.